guides-markdown/connect-local-nodes.md

```diff
@@ -0,0 +1,141 @@
+# Connect two local nodes
+
+```post-author
+Orfeas Stefanos Thyfronitis Litos
+```
+
+```post-description
+Start and manually connect two regtest nodes in a single script.
+```
+
+When we start two nodes in the regtest (loopback network, 127.0.0.1) with
+manually set ports, they don't get connected automatically because they don't
+know each other's port. We thus have to connect them manually. In this example
+we will fire up one SPV and one Full node and initiate a connection from the
+SPV side. The SPV node is given the address of the Full node and attempts to
+connect. If everything goes well, the two nodes connect. The Full node then
```

Comment:
Same here. Also, let's split the paragraph here when you start talking about transactions (easier to read smaller paragraphs and this is kind of another section). So "Next, we'll have the full node broadcast two transactions."

Reply:
(+1) I also rephrased the first sentence of the newly split paragraph to "To demonstrate that the connection was successful, we'll have full node..." to make clear why on earth we deal with txs.